apps/settings/style/icons.css

```diff
@@ -280,10 +280,6 @@ aside.connecting {
   background-position: 0 0;
 }
 
-.privacy-everythingME:before {
-  background-position: -3.2rem 0;
```

Comment:
The image still exists in the image asset - we can remove it if needed, but would need to get UX involved to generate the new assets I think?